modules/template/src/main/twirl/scaladex/view/project/artifact.scala.html

```diff
@@ -44,6 +44,9 @@
               <h3>
                 <a href="@artifactsUri(project.reference, artifact.artifactName, params.binaryVersion)">@artifact.artifactName</a>
                 <span class="artifact-version">@artifact.version</span>
+                @if(project.settings.deprecated) {
```

Comment:
We make a distinction between a deprecated and a deprecated artifact. A non-deprecated project can contain deprecated artifacts.

So here you should use `project.settings.artifactDeprecations.contains(artifact.artifactName)`